examples/chat/client/tasks.nim

```diff
@@ -190,6 +190,49 @@ proc addWalletPrivateKey*(name: string, privateKey: string, password: string)
   trace "task sent event to host", event=eventEnc, task
   asyncSpawn chanSendToHost.send(eventEnc.safe)
 
+proc addWalletSeed*(name: string, mnemonic: string, password: string,
```

Comment:
Styling: consider removing superfluous string type specifiers

Reply:
Like this?
```nim
proc addWalletSeed*(name, mnemonic, password, bip39Passphrase: string)
```

Reply:
Yes!

Reply:
Cool, I just wanted to make sure I understood you correctly. I often forget to take advantage of that shortcut, thanks for the tip!

Reply:
After trying to make this change, I remember now why I didn't do this perviously for tasks. When combining same-typed parameters in the signature *of a task proc*, the compiler barfs with this error:
```
/Users/emizzle/repos/status-im/nim-status/examples/chat/client/tasks.nim(193, 37) Error: initialization not allowed here
```
I really don't know why, but I'm assuming it has something to do with the `task` macro.